README.md

```diff
@@ -1,17 +1,25 @@
-# civil-disobedience-toolkit
+# Civil-Disobedience-Toolkit
 
 #Purpose
-The purpose of the civil disobedience toolkit is to gather a collection of open source tools that enable free speech and civil disobedience. Currently therer are several technological tools used to enable mass surveillance of peaceful protests as
-well as stifle dissent. Tools such as stingrays to capture cell phone data,  unmanned aerial vehicles (drones), surveillance
+The purpose of the Civil Disobedience Toolkit is to gather a collection of open source tools that enable free speech and civil disobedience. Currently there are several technological tools used to enable mass surveillance of peaceful protests as
+well as stifle dissent. Tools such as stingrays to capture cell phone data, unmanned aerial vehicles (drones), surveillance
 over social media by creating fake social media accounts to monitor political activities of activists on social media. 
 
 #Focus Areas
-1. Enabling secure communications - Allowing activists to use secure encrypted means of communication to prevent mass surveillanve
+1. Enabling secure communications - Allowing activists to use secure encrypted means of communication to prevent mass surveillance
+ * Vitual Private Networks (VPN)
+ * Orbot (Tor for Android)
+ * TextSecure (IOS)
+ * Wickr (IOS & Android)
+ * Silent Text (Android)
 2. Identifying countermeasures of mass surveillance - Identifying the tools and techniques of mass surveillance used to stifle dissent and 
-identifying effective tech countermeasures. 
+identifying effective tech countermeasures.
 3. Enabling organization and connectivity -Identify or create tools for activists to organize and broadcast their messages effectively and coordinate better
 4. Increasing resilience - Identifying tools and techniques that increase the likelihood that civil action will be successful and
 less prone to disruption due to targeting of key activists within an organization. 
+ * Remind activist to use 2FA with login accounts
+ * Educate those on the dangers of using open Wifi
+ * Maybe encourage burner phones if necessary
```

Comment:
This might not be cost-effective for some people. $30 for one person is someone's whole day of food, especially if you're an activist who lost their job *cough*.